### PR TITLE
Adding some proverbs

### PR DESCRIPTION
**Pull request by bastosmichael**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,3 +310,23 @@ Please tweet <@antjanus> if you do use it so I can see it in action!
...
+
+####Hofstadter's Law will tel you to always add more time than you think you need to a project because it will take longer, even when you take into account Hofstadter's Law.
+
+***
+
+####Long lasting code is written only when you pretend that it will be peer reviewed or maintained by a violent psycopath who knows where you live.
+
+***
+
+####Will the project be done on time? Will the project be under budget? Will the project be done properly? You can only ever honestly answer yes to two out of three of these questions for any given project.
```
Comment (AntJanus) [inline README.md]: not much of a proverb.

File: README.md [hunk 2]
```diff
@@ -310,3 +310,23 @@ Please tweet <@antjanus> if you do use it so I can see it in action!
...
+
+####Writing requirements based code and walking on water are both relatively easy to do when frozen.
+
+***
+
+####It takes twice as much intelligence to debug than to program, therefore you peer review because you can never truly be smart enough to debug your own code.
+
+***
+
+####Hofstadter's Law will tel you to always add more time than you think you need to a project because it will take longer, even when you take into account Hofstadter's Law.
```
Comment (AntJanus) [inline README.md]: Mispelling "tel"

**Comment by AntJanus** [comment]
I'll merge this in when you correct the mistakes and removed that one proverb :)

**Comment by bastosmichael** [comment]
@AntJanus done :)

**Pull request merged**